Timestamp: 2023-09-15T23:14:17.545Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:14:47.444Z
Caught exception: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
Exception origin: unhandledRejection
Stack trace:
MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
    at Function.Model.$wrapCallback (/home/container/node_modules/mongoose/lib/model.js:5095:32)
    at /home/container/node_modules/mongoose/lib/query.js:4510:21
    at /home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (/home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)
    at model.Query.exec (/home/container/node_modules/mongoose/lib/query.js:4509:10)
    at model.Query.Query.then (/home/container/node_modules/mongoose/lib/query.js:4592:15)

Timestamp: 2023-09-15T23:14:47.547Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:15:17.549Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:15:47.429Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:708:12
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Timestamp: 2023-09-15T23:15:47.429Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:736:10
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
    at runNextTicks (node:internal/process/task_queues:65:3)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)

Timestamp: 2023-09-15T23:15:47.430Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:759:10
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Timestamp: 2023-09-15T23:15:47.430Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:795:10
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Timestamp: 2023-09-15T23:15:47.433Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:830:10
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Timestamp: 2023-09-15T23:15:47.433Z
Caught exception: TypeError: Cannot read properties of undefined (reading 'length')
Exception origin: uncaughtException
Stack trace:
TypeError: Cannot read properties of undefined (reading 'length')
    at /home/container/server.js:853:10
    at /home/container/node_modules/mongoose/lib/model.js:5082:18
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Timestamp: 2023-09-15T23:15:47.445Z
Caught exception: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
Exception origin: unhandledRejection
Stack trace:
MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
    at Function.Model.$wrapCallback (/home/container/node_modules/mongoose/lib/model.js:5095:32)
    at /home/container/node_modules/mongoose/lib/query.js:4510:21
    at /home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (/home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)
    at model.Query.exec (/home/container/node_modules/mongoose/lib/query.js:4509:10)
    at model.Query.Query.then (/home/container/node_modules/mongoose/lib/query.js:4592:15)

Timestamp: 2023-09-15T23:15:47.551Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:16:17.553Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:16:47.445Z
Caught exception: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
Exception origin: unhandledRejection
Stack trace:
MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
    at Function.Model.$wrapCallback (/home/container/node_modules/mongoose/lib/model.js:5095:32)
    at /home/container/node_modules/mongoose/lib/query.js:4510:21
    at /home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (/home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)
    at model.Query.exec (/home/container/node_modules/mongoose/lib/query.js:4509:10)
    at model.Query.Query.then (/home/container/node_modules/mongoose/lib/query.js:4592:15)

Timestamp: 2023-09-15T23:16:47.554Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:17:17.555Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-15T23:17:47.445Z
Caught exception: MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
Exception origin: unhandledRejection
Stack trace:
MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
    at Function.Model.$wrapCallback (/home/container/node_modules/mongoose/lib/model.js:5095:32)
    at /home/container/node_modules/mongoose/lib/query.js:4510:21
    at /home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:32:5
    at new Promise (<anonymous>)
    at promiseOrCallback (/home/container/node_modules/mongoose/lib/helpers/promiseOrCallback.js:31:10)
    at model.Query.exec (/home/container/node_modules/mongoose/lib/query.js:4509:10)
    at model.Query.Query.then (/home/container/node_modules/mongoose/lib/query.js:4592:15)

Timestamp: 2023-09-15T23:17:47.557Z
Caught exception: AbortError: The user aborted a request.
Exception origin: unhandledRejection
Stack trace:
AbortError: The user aborted a request.
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

Timestamp: 2023-09-22T05:15:16.326Z
Caught exception: FetchError: request to https://discord.com/api/v9/channels/931064578580824075/messages/962784598947946606 failed, reason: getaddrinfo EAI_AGAIN discord.com
Exception origin: unhandledRejection
Stack trace:
FetchError: request to https://discord.com/api/v9/channels/931064578580824075/messages/962784598947946606 failed, reason: getaddrinfo EAI_AGAIN discord.com
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async MessageManager.edit (/home/container/node_modules/discord.js/src/managers/MessageManager.js:132:15)

Timestamp: 2023-09-22T05:15:16.420Z
Caught exception: FetchError: request to https://discord.com/api/v9/channels/931064578580824075/messages/962784619227385856 failed, reason: getaddrinfo EAI_AGAIN discord.com
Exception origin: unhandledRejection
Stack trace:
FetchError: request to https://discord.com/api/v9/channels/931064578580824075/messages/962784619227385856 failed, reason: getaddrinfo EAI_AGAIN discord.com
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async MessageManager.edit (/home/container/node_modules/discord.js/src/managers/MessageManager.js:132:15)

Timestamp: 2023-09-22T05:15:16.422Z
Caught exception: FetchError: request to https://discord.com/api/v9/channels/923061932272017449/messages failed, reason: getaddrinfo EAI_AGAIN discord.com
Exception origin: unhandledRejection
Stack trace:
FetchError: request to https://discord.com/api/v9/channels/923061932272017449/messages failed, reason: getaddrinfo EAI_AGAIN discord.com
    at RequestHandler.execute (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:201:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async RequestHandler.push (/home/container/node_modules/discord.js/src/rest/RequestHandler.js:51:14)
    at async TextChannel.send (/home/container/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js:176:15)

